### PR TITLE
Add rds to laa-assure-hmrc-data-uat

### PR DESCRIPTION
**Pull request by kmahern**

### CODE REVIEW

**Review by jsugarman — COMMENTED** [review]
See a couple of comments. regarding the removal of read-replicas I think we should ask CP for the reason they add it by default. I have read that they can provide disaster recovery failover for example - albeit this is not important for UAT

File: namespaces/live.cloud-platform.service.justice.gov.uk/laa-assure-hmrc-data-uat/resources/rds-postgresql.tf [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+  providers = {
+    # Can be either "aws.london" or "aws.ireland"
+    aws = aws.london
+  }
+}
+
+resource "kubernetes_secret" "rds" {
+  metadata {
+    name      = "rds-postgresql-instance-output"
```
Comment (jsugarman) [inline namespaces/live.cloud-platform.service.justice.gov.uk/laa-assure-hmrc-data-uat/resources/rds-postgresql.tf:63]: can we just make this name `rds-instance-output`

File: namespaces/live.cloud-platform.service.justice.gov.uk/laa-assure-hmrc-data-uat/resources/rds-postgresql.tf [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    database_username     = module.rds.database_username
+    database_password     = module.rds.database_password
+    rds_instance_address  = module.rds.rds_instance_address
+    access_key_id         = module.rds.access_key_id
+    secret_access_key     = module.rds.secret_access_key
+  }
+  /* You can replace all of the above with the following, if you prefer to
+     * use a single database URL value in your application code:
+     *
+     * url = "postgres://${module.rds.database_username}:${module.rds.database_password}@${module.rds.rds_instance_endpoint}/${module.rds.database_name}"
```
[the remark this answers is not in the document]
Comment (jsugarman) [reply namespaces/live.cloud-platform.service.justice.gov.uk/laa-assure-hmrc-data-uat/resources/rds-postgresql.tf:79]: 🤔 maybe we don't need for now. can always add later

File: namespaces/live.cloud-platform.service.justice.gov.uk/laa-assure-hmrc-data-uat/resources/rds-postgresql.tf (hunk 1 above)
Comment (jsugarman) [reply namespaces/live.cloud-platform.service.justice.gov.uk/laa-assure-hmrc-data-uat/resources/rds-postgresql.tf:63]: scratch that as automatic naming

File: namespaces/live.cloud-platform.service.justice.gov.uk/laa-assure-hmrc-data-uat/resources/rds-postgresql.tf [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  #     value        = "0"
+  #     apply_method = "pending-reboot"
+  #   }
+  # ]
+
+  # use "allow_major_version_upgrade" when upgrading the major version of an engine
+  allow_major_version_upgrade = "false"
+
+  # Enable auto start and stop of the RDS instances during 10:00 PM - 6:00 AM for cost saving, recommended for non-prod instances
+  # enable_rds_auto_start_stop  = true
```
[the remark this answers is not in the document]
Comment (kmahern) [reply namespaces/live.cloud-platform.service.justice.gov.uk/laa-assure-hmrc-data-uat/resources/rds-postgresql.tf]: 👍

**Review by sj-williams — APPROVED** [review]

**Pull request merged**